### PR TITLE
Blog App - Views Milestone

### PR DESCRIPTION
**Pull request by lily4178993**
## Description:

This pull request addresses the "Blog app - Views" milestone. The required view files have been updated with the design elements from the sneak peek wireframes. Additionally, basic styles have been applied to enhance the visual presentation of each view file.

## Changes:

- Updated views based on [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#project-requirements)
- Applied basic styles to enhance the visual appearance

## Testing:

- Verified that views render as expected
- Checked responsiveness on different screen sizes

---
_**Please review and provide feedback at your earliest convenience.**_

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+    <% if @posts.any? %>
+      <div class="grid grid-cols-1">
+        <% @posts.each do |post| %>
+          <%= link_to user_post_path(@user, post) do %>
+            <%= render partial: "shared/post_card_preview", locals: { post: post } %>
+          <% end %>
+        <% end %>
+      </div>
+      <div class="w-fit mx-auto my-8">
+        <%= link_to "See all posts" , user_posts_path(@user),
```
Comment (zeff96) [inline app/views/users/show.html.erb:17]: - [ ] Hello, @lily4178993! 👋 Great job on implementing the link that navigates to the post#index of a specific user. 🌟 However, I've noticed a slight deviation from the project requirements. 🚧 In your current implementation, when I visit this path, I can only see the three most recent posts. 📆 This isn't the intended behavior; the path should display all posts for the given user. 📚 I've attached an image to help you visualize the expected structure of this page. 🖼️ Let's make sure it aligns with the project requirements! 🤝

postindex

**Review by zeff96 — CHANGES_REQUESTED** [review]
### Status: CHANGES REQUIRED ♻️
Hi @lily4178993 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
:heavy_check_mark: Linters passing
:heavy_check_mark: Views added

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (lily4178993) [reply app/views/users/show.html.erb:17]: Hi @zeff96 ,
Thank you for the review and feedback!

Comment (Whoistolu) [reply app/views/users/show.html.erb:17]: - Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @lily4178993 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Thank you, @Whoistolu , for the review and feedback!

**Pull request merged**